javalib/src/main/scala/java/nio/file/attribute/UserPrincipalNotFoundException.scala

```diff
@@ -0,0 +1,9 @@
+package java.nio.file.attribute
+
+import java.io.IOException
+
+class UserPrincipalNotFoundException(name: String) extends IOException {
+  def getName(): String = name
+
+  override def getMessage(): String = s"Lookup for $name failed"
```

Comment:
This line should be deleted.  SN javalib code attempts to emulate Java 8 JVM as closely as
feasible.  I checked the JVM and its getMessage() does a simple pass through. 
Such an override would be a best-practice in other areas of SN, say posixlib.

Reply:
Done.